### PR TITLE
[plugin-accessibility] Add ability to report a11y violations individu…

### PR DESCRIPTION
**Pull request by uarlouski**
…ally

### CODE REVIEW

File: vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/AccessibilitySteps.java [hunk 1]
```diff
@@ -126,14 +131,30 @@ public void performAccessibilityScan(ExamplesTable checkOptions)
...
+        {
+            violations.stream().map(v -> "[" + idGetter.apply(v) + "] " + messageGetter.apply(v))
+                    .forEach(softAssert::recordFailedAssertion);
+
+            if (!violations.isEmpty())
+            {
+                return;
+            }
+        }
+        softAssert.assertThat(String.format(format, args), (long) violations.size(), equalTo(0L));
```
Comment (valfirst) [inline vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/AccessibilitySteps.java:155]:
```suggestion
        else
        {
            softAssert.assertThat(String.format(format, args), (long) violations.size(), equalTo(0L));
        }
```

Comment (valfirst) [inline vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/AccessibilitySteps.java:155]:
```suggestion
        }
        else
        {
            softAssert.assertThat(String.format(format, args), (long) violations.size(), equalTo(0L));
        }
```

Comment (uarlouski) [reply vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/AccessibilitySteps.java]: fixed, but it was intentional to perform this soft assert just to keep step non empty when there are no violations

Comment (uarlouski) [reply vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/AccessibilitySteps.java]: merge pls :)

**Review by valfirst — APPROVED** [review]

**Pull request merged**